config/rbac/role.yaml

```diff
@@ -37,7 +37,7 @@ rules:
   - patch
   - delete
 - apiGroups:
-  - cluster.k8s.io
+  - cluster.sigs.k8s.io
```

Comment:
This is likely fine, but the migration tool will still need rbac permissions to modify objects in the `cluster.k8s.io` api group.